core/http/src/header/forwarded_protocol.rs

```diff
@@ -0,0 +1,38 @@
+use std::{
+    fmt::{self, Display},
+    str::FromStr,
+};
```

Comment:
```suggestion
use std::fmt;
```

Then use `fmt::Display` and `std::str::FromStr` later.